src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -56,6 +60,15 @@ class CassandraJournal extends AsyncWriteJournal with CassandraRecovery with Cas
 
   session.execute(writeConfig, CassandraJournalConfig.TargetPartitionProperty, config.targetPartitionSize.toString)
 
+  // TODO: Figure out how to sensibly run the merging process.
+  Cluster(context.system).join(Cluster(context.system).selfAddress)
```

Comment:
This is not ideal, but it might be a viable option. Not happy about the hacky cluster join though. Any ideas how to manage the stream merging component lifecycle reliably?